src/server/scripts/Northrend/Ulduar/Ulduar/boss_razorscale.cpp

```diff
@@ -15,15 +15,16 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-/// @todo Harpoon chain from 62505 should not get removed when other chain is applied
-
 #include "ScriptMgr.h"
 #include "GameObject.h"
 #include "GameObjectAI.h"
 #include "InstanceScript.h"
 #include "Map.h"
 #include "MotionMaster.h"
+#include "MoveSplineInit.h"
 #include "ObjectAccessor.h"
+#include "PassiveAI.h"
+#include "PathGenerator.h"
```

Comment:
Should be changed to #include <G3D/Vector3.h> right?
And what the rule to put this includes? alphabetical order?